src/cli/commands/sonarcloud/set-token.ts

```diff
@@ -0,0 +1,55 @@
+import read from "read"
```

Comment:
Not entirely sure if it makes sense to set up CLI commands for SonarCloud if we're essentially only making the `set-token` command available as a CLI command. If we have functionality for running `npx @capraconsulting/cals-cli sonarcloud fetch-metrics <github-repository>` (or similar) it may make more sense, but in it's current form we may want to skip CLI config for SonarCloud altogether. What do you think?

Reply:
I agree, it will not be used anyways unless we implement more methods